README.md

```diff
@@ -5,6 +5,12 @@ The main purpose of this module is to provide a PuppetDB import source
 for Icinga Director. As of this writing it has not even a GUI or CLI
 component, but this may change in future.
 
+To install, please refer to our documentation. [Installation and configuration](doc/01-Installation.md)
+Note: Some steps require you to be familiar with Puppet/PuppetDB. 
+
+
+==========
+
+Screnshot:
```

Comment:
I couldn't lookup at the squash before, but following your advice, i searched for it.  Pretty useful to clean up messy history.  I will surely use it next time.   I'm still quite beginner using git and using mostly the most basic feature of it.   Working on getting better as I go. :)